moto/sagemaker/__init__.py

```diff
@@ -0,0 +1,6 @@
+from __future__ import unicode_literals
+from .models import sagemaker_backends
+
+sagemaker_backend = sagemaker_backends["us-east-1"]
+mock_sagemaker = sagemaker_backend.decorator
+mock_sagemaker_deprecated = sagemaker_backend.deprecated_decorator
```

Comment:
Let's remove the `deprecated` decorator - we're moving away from those anyway

Reply:
Done